feat: use user local data dir for lsp daemon files (#1238)
biomejs/biome: website/src/content/docs/guides/integrate-in-editor.mdx

### Daemon logs

> The Biome daemon saves logs in your file system. Logs are store in a folder called `biome-logs`. You can find this folder in the biome cache directory, residing at `~/.cache/biome` on Linux, `C:\Users\<UserName>\AppData\Local\biomejs\biome\cache` on Windows, `/Users/<UserName>/Library/Caches/dev.biomejs.biome` on macOS, and the system's temporary directory on other operating systems.

Reviewer: ```suggestion
The Biome daemon saves logs in your file system. Logs are stored in a folder called `biome-logs`. The path of  this folder changes based on your operative system:
-  Linux: `~/.cache/biome`;
-  Windows: `C:\Users\<UserName>\AppData\Local\biomejs\biome\cache`
-  macOS: `/Users/<UserName>/Library/Caches/dev.biomejs.biome`

For other operative systems, you can find the folder in the system's temporary directory.
```
